src/components/application_manager/rpc_plugins/sdl_rpc_plugin/include/sdl_rpc_plugin/commands/mobile/delete_interaction_choice_set_request.h

```diff
@@ -43,6 +46,8 @@ namespace app_mngr = application_manager;
 
 namespace commands {
 
+using SentRequestsSet = std::set<std::uint32_t>;
```

Comment:
@IGapchuk why don't you use typedef instead?

Reply:
@AKalinich-Luxoft Changed in e32b004ae4d7e84e153d6b8082cb3c1a800a41ec.